libs/app-example-forms/src/lib/services/create-appointment-form.service.ts

```diff
@@ -0,0 +1,30 @@
+import { inject, Injectable } from '@angular/core'
+import { FormBuilder, FormGroup, Validators } from '@angular/forms'
+
+@Injectable()
+export class CreateAppointmentFormService {
```

Comment:
```suggestion
export class AppointmentEditFormService {
```

A better way to name form service is to name it `Edit` instead of `Create`. Because you will use it for both create and update. Also, see the `AbstractFormService` utilization. It will help you write some less codes.